### DISCUSSION TITLE
i cant find my profile when i sign in

### DISCUSSION THREAD

**Discussion by ghost** [community/community#102860] [New to GitHub]
Hello my friends I have an account on GitHub and this account is raised on it all my work and work and I am very tired to be able to be this account and if a problem happened in my computer and when I solved it it may delete all the registration files in the search engine and when I wanted to register again to enter my account again I encountered a problem and when registering he took me to another account that is not my real account and here is my real account https://github.com/abdalrhmanfolly/ this is his link And I hope that I can recover my account in any realistic way such as registering the face or verifying the identity card

**Comment by mw-xyz** [comment]
If you remember the email you used, you can request to reset your password for it. Try look through your mailboxes for any emails form GitHub.
